python/ray/worker.py

```diff
@@ -1591,6 +1592,10 @@ def init(redis_address=None,
         Exception: An exception is raised if an inappropriate combination of
             arguments is passed in.
     """
+    # Add the use_raylet option for backwards compatibility.
+    if not use_raylet:
+        raise Exception("The use_raylet argument is no longer supported.")
```

Comment:
maybe we could improve the message to communicate that (a) it's being deprecated and (b) only raylet mode is supported? E.g., "The use_raylet argument is deprecated. Only use_raylet=True mode is supported."

Reply:
Another option is to default to None, and if True then raise DeprecationWarning and if False raise an exception with a deprecation message similar to what Alexey suggested?